### ISSUE TITLE
improve message when docker is not installed for none

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#7166]
when docker is not installed for none, instead of crashing, we could tell user nicely, please install docker
```
medya@none-experiment-instance-1-medya:~$ sudo minikube start --vm-driver=none
😄  minikube v1.8.2 on Ubuntu 18.04
✨  Using the none driver based on user configuration
🤹  Running on localhost (CPUs=2, Memory=7470MB, Disk=9749MB) ...
💣  Unable to start VM. Please investigate and run 'minikube delete' if possible: creating host: create: precreate:
 exec: "docker": executable file not found in $PATH
😿  minikube is exiting due to an error. If the above message is not useful, open an issue:
👉  https://github.com/kubernetes/minikube/issues/new/choose
```

**Comment by afbjorklund** [comment]
I still think that this one is worse: https://github.com/kubernetes/minikube/issues/5549

**Comment by afbjorklund** [comment]
Possibly related to #6402

**Comment by zerefwayne** [comment]
@priyawadhwa @afbjorklund I think I can work on this one.

**Comment by priyawadhwa** [comment]
@zacharysarah great, I'll assign you to the issue. Let me know if you have any questions.

**Comment by zerefwayne** [comment]
@priyawadhwa @medyagh

1. How do I reproduce the error as I have docker installed on Ubuntu 18.04? Is there any alternative to completely removing docker to solve this problem?
2. I couldn't find the exact location in code from where the error message is being generated even after using grep.

**Comment by afbjorklund** [comment]
zerefwayne : you can just rename it temporarily, and move it back again when done

`sudo mv /usr/bin/docker /usr/bin/docker.orig`

**Comment by prasadkatti** [comment]
this issue should be closed since it is fixed by #7388
